Dockerfile

```
@@ -27,7 +27,7 @@ ENV PYTHONPATH="${PYTHONPATH}":/python/pkgs
 COPY --from=py-deps /python/__pypackages__/3.10/lib /python/pkgs
 COPY --from=py-deps /python/__pypackages__/3.10/bin/* /bin/
 
-RUN apt-get update && apt-get -y upgrade && rm -rf /var/lib/apt/lists/* && \
+RUN apt-get update && apt-get -y install wget && apt-get -y upgrade && rm -rf /var/lib/apt/lists/* && \
```

Prefer `curl` and upgrade first:
```suggestion
RUN apt-get update && apt-get -y upgrade && apt-get -y install curl && rm -rf /var/lib/apt/lists/* && \
```